modules/ROOT/pages/upgrade.adoc

```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
+Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
 
-Refer to the xref:release-notes/pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) release notes for information about upgrading Anypoint Platform PCE to the most recent version.
+Refer to the xref:release-notes::pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform PCE release notes for information about upgrading Anypoint Platform PCE to the most recent version.
 
 See xref:1.6@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.6.1. See xref:1.7@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.7.1.
 
-[WARNING]
-====
-Before upgrading:
+If you are using AWS, obtain the keys used to run the AWS provisioner. These are required for access EC2, VPC, and S3.
```

Comment:
This is only for installations. It's not necessary to run the AWS provisioner for updates.

Reply:
Removed - thank you!